tv_apps/tv-epg/js/epg.js

```diff
@@ -263,8 +234,7 @@
       if (prevPrograms && prevPrograms[row] &&
           programElement !== prevPrograms[row].element) {
         programElement = prevPrograms[row].element;
-        programElement.querySelector('.background-progress').style.transform =
-          'scaleX(1)';
+        programElement.fillProgress();
```

Comment:
May we fill progress automatically? Do we really need to call a function to fill it?